README.md

```
@@ -4,36 +4,37 @@
-- Include any custom script you need to make your presentation clearer through eg animation.
+Slipshow improves on traditional slide-based presentations in the following ways:
+- Scroll on a slip instead of changing slides, making content flow continuously
+- Record hand-drawings when preparing your talk, edit them, and have them replayed during the presentation
+- Include and execute custom scripts by steps, to make your presentation clearer through e.g. animation or interaction.
 
-Slipshow compiles files written in an extension of markdown, to a standalone html file viewable offline in any web browser.
-
-If you want to know more, go to the [official website](https://slipshow.org)! This page is about the development of Slipshow.
+Slipshow compiles files written in an extension of [Markdown](https://en.wikipedia.org/wiki/Markdown), to a standalone HTML file viewable offline in any web browser.
 
 ##  Tutorial, API and Documentation
```

markdownlint-cli2 (0.23.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)